plugins/woocommerce/src/Admin/Features/Features.php

```diff
@@ -358,4 +370,26 @@ private function register_internal_class_aliases() {
 			class_alias( $new_class, $orig_class );
 		}
 	}
+
+	/**
+	 * Check if we're in an admin context where features should be loaded.
+	 *
+	 * @return boolean
+	 */
+	private static function should_load_features() {
+		if ( class_exists( 'WC_Unit_Tests_Bootstrap' ) ) {
+			// Always load features in unit tests.
```

Comment:
I don't like the idea of production code actively checking for a unit testing context... is this really needed? If there are tests that get broken when admin features aren't loaded, maybe these tests should be adjusted instead.

Alternatively, the `is_admin()` check below could be replaced with `wc_get_container()->get(LegacyProxy::class)->call_function('is_admin')`, and then the function can be mocked to always return true in the tests where that's needed.

Reply:
Thanks for the feedback! @Konamiman 🙏  The reason for the check is that some tests like `Payment_Gateways_V2`, which test the REST API endpoints, require loading admin features to return the correct data. We could potentially adjust these tests, but it might change their expected behavior and make them less useful, so I didn't want to go down that route.

> Alternatively, the is_admin() check below could be replaced with wc_get_container()->get(LegacyProxy::class)->call_function('is_admin'), and then the function can be mocked to always return true in the tests where that's needed.

I gave it a try, but it didn't work well. The admin features are loaded in the `init` hook, which is loaded during the WP testing environment initialization, so we need to mock `is_admin` before the environment is initialized. However, I think that would affect all tests, not just those needing admin features.

Reply:
As your comment suggests below, I think the best approach is to add a filter. It allows the extension to load admin features in different contexts if needed and also allows us to force load admin features in the testing environment. I'll add a filter and hook it into `tests/legacy/bootstrap.php` to load admin features in the testing environment. 

Let me know if you have any other suggestions or concerns. 🙂

Reply:
@Konamiman, Could you please review the changes I made when you get a chance? Thanks! 🙂